Improvements to Flux GitHub Action (#3474)
fluxcd/flux2: action/README.md

          version: 0.32.0
```

> You can also authentication to the GitHub API using a secret

Reviewer: ```suggestion
You can also authentication to the GitHub API using an [encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets):
```

Might help a bit with users switching from a Kubernetes to GitHub (Action) context.

Reviewer 2: I think that might confuse users actually because the value for `secrets.GITHUB_TOKEN` is pre-filled by GitHub Actions. Might be better to link to this instead? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reviewer: That's an even better suggestion indeed.

Author: Adjusted the documentation to include the second link and provide a bit more context.

> You can also authentication to the GitHub API using GitHub Action's `GITHUB_TOKEN`. For more information on this secrets, please see [about the github token secret](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret)

Reviewer 2: Just a nit here and we can merge :)
```suggestion
You can also authenticate against the GitHub API using GitHub Actions' `GITHUB_TOKEN` secret.
For more information, please [read about the GitHub token secret](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret).
```
